src/pymortests/algorithms/svd_va.py

```diff
@@ -31,14 +30,15 @@ def test_method_of_snapshots(vector_array, method):
 
     B = A.copy()
     with log_levels({"pymor.algorithms": "ERROR"}):
-        U, s, Vh = method(A)
+        U, s, Vh = method(A, rtol=4e-8)
```

Comment:
Could you add that this is current default rtol from the methods instead of a magic number and reference it below? I forgot to document that.

Reply:
Is it actually important that this is the default tolerance? The test should work with other tolerances as well. Does the test need to be adapted, when the default tolerance is changed?

Reply:
The info is only important insofar it's just a random magic number.